packetbeat/beater/packetbeat.go

```diff
@@ -135,16 +126,76 @@ func (pb *Packetbeat) Setup(b *beat.Beat) error {
 		return fmt.Errorf("Initializing sniffer failed: %v", err)
 	}
 
+	return nil
+}
+
+func (pb *Packetbeat) Run(b *beat.Beat) error {
+	defer func() {
+		if service.WithMemProfile() {
+			logp.Debug("main", "Waiting for streams and transactions to expire...")
+			time.Sleep(time.Duration(float64(protos.DefaultTransactionExpiration) * 1.2))
+			logp.Debug("main", "Streams and transactions should all be expired now.")
+		}
+
+		// TODO:
```

Comment:
no idea yet. Just copy'n pasted from original location. Gotta check if it can be removed (another PR).